schemas/device.yml

```diff
@@ -23,6 +23,9 @@
       type: text
       description: >
         Device vendor information.
+      multi_fields:
```

Comment:
This change would affect every single event we have in Beats at the moment and I'm very hesitant to make this change.